### ISSUE TITLE
Setting dual thresholds with modkit

### ISSUE THREAD

**Issue by pwh124** [nanoporetech/modkit#13]
Continuing from https://github.com/epi2me-labs/modbam2bed/issues/57

We traditionally set thresholds for both unmethylated (<20%) and methylated (>80%). This throws out ambiguous calls (20-80%). This approach is supported for methylation called by megalodon in a recent publication (https://www.nature.com/articles/s41467-021-23778-6). 

For modifications like 5mC, there usually are not a lot of ambiguous calls, but calling everything <80% probability as "unmethylated" does not seem prudent.

Previously, to perform this dual thresholding, I would use modbam2bed with the following command:

```
 modbam2bed -t {threads} \
		-e \
		-m 5mC \
		--cpg \
		-a 0.20 -b 0.80 \
		{input.genome} {input.modbam} > {output}
```
Since modbam2bed is being deprecated, how would I perform this same type of command with modkit?

**Comment by ArtRand** [comment]
Hey @pwh124, 

This feature is slated for the next release scheduled to come out next week. I'd be willing to get you a release candidate ASAP if you're OK with testing something that may not be perfect yet.

**Comment by pwh124** [comment]
Hi @ArtRand!

This feature coming out next week works for me, I am still processing sequencing reads prior to modification calling.  I was just trying to get ahead in my pipeline.

Thanks!

**Comment by ArtRand** [comment]
Actually, after reading your use case a little more closely. You should be able to do what you want with the released version, specifying `--filter-threshold 0.8` will require a base modification call has a probability of being methylated >= 80% or <= 20%. 

The feature coming out next week will let you have asymetric thresholds.

**Comment by pwh124** [comment]
Ohhh okay. This makes sense - I will try it out!

**Comment by pwh124** [comment]
@ArtRand - modkit seems to be working great for this. Thank you for resolving my confusion.

Just for reference, I am running this command:

```
~/software/modkit_0.1.5/modkit pileup --log-filepath mod_calls/mod_forebrain.log \
-t24 \
--seed 24 \
--filter-threshold 0.8 \
--cpg \
--ref ~/references/mm39/mm39.fa \
--sampling-frac 0.1 \
--only-tabs \
alignments/mod.bam mod_calls/mods.bed
```

**Comment by ArtRand** [comment]
That's great!

**Comment by ArtRand** [comment]
@pwh124 ,

Multiple thresholds were released in [0.1.6](https://github.com/nanoporetech/modkit/releases/tag/v0.1.6) and [documentation](https://nanoporetech.github.io/modkit/filtering_details.html) has been added. If you think this is clear enough and solves your use case would you mind closing this issue? Thanks.

**Comment by pwh124** [comment]
Looks good to me - thanks!
